src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -48,7 +48,7 @@
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7" Version="3.0.452" />
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.22" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="4.0.1" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.8" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.8.1" />
```

Comment:
I'm not positive, but I'm guessing they will want to stick to patch version increments at the smallest considering the current format.

Also, we should talk to someone more familiar with the standard extension model how we can ensure this gets released before the host goes out.

Reply:
Yes, just increment the patch version.